src/utils/urls.js

```diff
@@ -0,0 +1,17 @@
+export function extractBugId(url) {
+  if (url.includes('bugzilla')) {
+    const match = /id=(\d+)/.exec(url);
+    if (match) {
+      return match[1];
+    }
+    return url;
+  }
+  if (url.includes('github')) {
+    const match = /(issues|pull)\/(\d+)/.exec(url);
+    if (match) {
+      return `#${match[2]}`;
```

Comment:
I think returning something like `mozilla/fenix#1234` would be better for GitHub issues (more idiomatic).

Reply:
This makes the links long and breaks the UI, but I can make each of those on their own row to allow for longer URL text...

![Screenshot_2020-10-16 storage_stats_cache_bytes GLAM](https://user-images.githubusercontent.com/1106/96285068-4d836d00-0f93-11eb-920d-76ed972576b0.png)